src/lib/events.js

```diff
@@ -3,6 +3,7 @@ export const VIEWER_EVENT = {
     default: 'viewerevent', // The default viewer event.
     download: 'download', // Begin downloading the file.
     error: 'error', // When an error occurs.
+    imageZoomClick: 'imagezoomclick', // When image is clicked directly to zoom/reset
```

Comment:
Baking the file type into the event name doesn't seem very scalable. It should be available as a separate property on the event.